exercises/practice/grade-school/test/grade_school_tests.erl

```diff
@@ -53,24 +92,66 @@
     S5 = grade_school:add("Alex", 2, S4),
     S6 = grade_school:add("Jim", 3, S5),
     S7 = grade_school:add("Charlie", 1, S6),
-    {"Student names with grades are displayed "
-     "in the same sorted roster",
+    {"Students are sorted by grades and then "
+     "by name in the roster",
      ?_assertEqual(["Alex", "Anna", "Barb", "Charlie", "Jim",
 		    "Peter", "Zoe"],
 		   lists:sort(grade_school:get(S7)))}.
 
-'6_grade_returns_the_students_in_that_grade_in_alphabetical_order_test_'() ->
+'10_grade_is_empty_if_no_students_in_the_roster_test_'() ->
+    S0 = grade_school:new(),
+    {"Grade is empty if no students in the "
+     "roster",
+     ?_assertEqual([], lists:sort(grade_school:get(1, S0)))}.
+
+'11_grade_is_empty_if_no_students_in_that_grade_test_'() ->
+    S0 = grade_school:new(),
+    S1 = grade_school:add("Peter", 2, S0),
+    S2 = grade_school:add("Zoe", 2, S1),
+    S3 = grade_school:add("Alex", 2, S2),
+    S4 = grade_school:add("Jim", 3, S3),
+    {"Grade is empty if no students in that "
+     "grade",
+     ?_assertEqual([], lists:sort(grade_school:get(1, S4)))}.
+
+'12_student_not_added_to_same_grade_more_than_once_test_'() ->
+    S0 = grade_school:new(),
+    S1 = grade_school:add("Blair", 2, S0),
+    S2 = grade_school:add("James", 2, S1),
+    S3 = grade_school:add("James", 2, S2),
+    S4 = grade_school:add("Paul", 2, S3),
+    {"Student not added to same grade more "
+     "than once",
+     ?_assertEqual(["Blair", "James", "Paul"],
+		   lists:sort(grade_school:get(2, S4)))}.
+
+'13_student_not_added_to_multiple_grades_test_'() ->
+    S0 = grade_school:new(),
+    S1 = grade_school:add("Blair", 2, S0),
+    S2 = grade_school:add("James", 2, S1),
+    S3 = grade_school:add("James", 3, S2),
+    S4 = grade_school:add("Paul", 3, S3),
+    % io:format("~p~n", [S4]),
```

Comment:
I assume this comment can be deleted?

Reply:
Yes. it can. I have missed it after changing `grade-school`